src/Raven.Studio/typescript/widgets/virtualGrid/virtualRow.ts

```diff
@@ -88,15 +88,11 @@ class virtualRow {
             
             if (!this._disableStripes) {
                 // Update the "even" status. Used for striping the virtual rows.
-                const newEvenState = rowIndex % 2 === 0;
-                const hasChangedEven = this._even !== newEvenState;
-                if (hasChangedEven) {
-                    this._even = newEvenState;
-                    if (this._even) {
-                        this.element.addClass("even");
-                    } else {
-                        this.element.removeClass("even");
-                    }
+                this._even = rowIndex % 2 === 0;
```

Comment:
what's the current purpose of even field? Why can't we leave previous logic? Right now we add/remove classes no matter what. Let's talk about this, ok?